cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/field/Range.java

```diff
@@ -38,20 +39,67 @@
 
   /**
    * Create a range with begin and end.
-   * @param begin the fields forming the beginning of the range
+   *
+   * @param begin the fields forming the beginning of the range, empty to read from the start
```

Comment:
does begin and end have to a complete primary key? Or a partial primary key? A prefix? What if it is not? Will that cause errors/exceptions?

Reply:
It has to be prefix or complete primary key, added that to javadoc